datafusion/sqllogictest/test_files/tpch/q8.slt.part

```diff
@@ -61,34 +61,35 @@ Sort: all_nations.o_year ASC NULLS LAST
 ------SubqueryAlias: all_nations
 --------Projection: date_part(Utf8("YEAR"), orders.o_orderdate) AS o_year, lineitem.l_extendedprice * (Decimal128(Some(1),20,0) - lineitem.l_discount) AS volume, n2.n_name AS nation
-----------Inner Join: n1.n_regionkey = region.r_regionkey
-------------Projection: lineitem.l_extendedprice, lineitem.l_discount, orders.o_orderdate, n1.n_regionkey, n2.n_name
---------------Inner Join: supplier.s_nationkey = n2.n_nationkey
-----------------Projection: lineitem.l_extendedprice, lineitem.l_discount, supplier.s_nationkey, orders.o_orderdate, n1.n_regionkey
-------------------Inner Join: customer.c_nationkey = n1.n_nationkey
---------------------Projection: lineitem.l_extendedprice, lineitem.l_discount, supplier.s_nationkey, orders.o_orderdate, customer.c_nationkey
-----------------------Inner Join: orders.o_custkey = customer.c_custkey
-------------------------Projection: lineitem.l_extendedprice, lineitem.l_discount, supplier.s_nationkey, orders.o_custkey, orders.o_orderdate
---------------------------Inner Join: lineitem.l_orderkey = orders.o_orderkey
-----------------------------Projection: lineitem.l_orderkey, lineitem.l_extendedprice, lineitem.l_discount, supplier.s_nationkey
-------------------------------Inner Join: lineitem.l_suppkey = supplier.s_suppkey
---------------------------------Projection: lineitem.l_orderkey, lineitem.l_suppkey, lineitem.l_extendedprice, lineitem.l_discount
-----------------------------------Inner Join: part.p_partkey = lineitem.l_partkey
-------------------------------------Projection: part.p_partkey
---------------------------------------Filter: part.p_type = Utf8("ECONOMY ANODIZED STEEL")
-----------------------------------------TableScan: part projection=[p_partkey, p_type], partial_filters=[part.p_type = Utf8("ECONOMY ANODIZED STEEL")]
-------------------------------------TableScan: lineitem projection=[l_orderkey, l_partkey, l_suppkey, l_extendedprice, l_discount]
---------------------------------TableScan: supplier projection=[s_suppkey, s_nationkey]
-----------------------------Filter: orders.o_orderdate >= Date32("9131") AND orders.o_orderdate <= Date32("9861")
-------------------------------TableScan: orders projection=[o_orderkey, o_custkey, o_orderdate], partial_filters=[orders.o_orderdate >= Date32("9131"), orders.o_orderdate <= Date32("9861")]
-------------------------TableScan: customer projection=[c_custkey, c_nationkey]
---------------------SubqueryAlias: n1
-----------------------TableScan: nation projection=[n_nationkey, n_regionkey]
-----------------SubqueryAlias: n2
-------------------TableScan: nation projection=[n_nationkey, n_name]
-------------Projection: region.r_regionkey
---------------Filter: region.r_name = Utf8("AMERICA")
-----------------TableScan: region projection=[r_regionkey, r_name], partial_filters=[region.r_name = Utf8("AMERICA")]
+----------Projection: lineitem.l_extendedprice, lineitem.l_discount, orders.o_orderdate, n2.n_name
```

Comment:
🤔  I am surprised to see the plan change here. I don't expect anything to change.

I will try and study what changed later and figure out what we might be able to change

Reply:
I am very unsure about this one. CI was being very unstable so I can't say if it's related to this change at all ... for the life of me I couldn't see any reason why it would have been. I didn't go to the length of bisecting the changes though to see if I could narrow it down as I've got a bit too much on my plate atm as it is.